jetty-server/src/main/java/org/eclipse/jetty/server/handler/ContextHandler.java

```diff
@@ -230,10 +231,10 @@ public static void setServerInfo(String serverInfo)
 
     public enum Availability
     {
-        UNAVAILABLE, STARTING, AVAILABLE, SHUTDOWN,
+        STOPPED, STARTING, AVAILABLE, UNAVAILABLE, SHUTDOWN,
```

Comment:
Can we have a comment about the meanings of the different states, particularly UNAVAILABLE vs STOPPED, which is a new state.